gensim/test/test_atmodel.py

```diff
@@ -436,7 +436,7 @@ def testPasses(self):
             for test_rhot in test_rhots:
                 model.update(corpus, author2doc)
 
-                msg = ", ".join(map(str, [passes, model.num_updates, model.state.numdocs]))
+                msg = ", ".join(str(x) for x in [passes, model.num_updates, model.state.numdocs])
```

Comment:
How about `msg = "%d, %d, %d" % (passes, model.num_updates, model.state.numdocs)`?
This is more simple.

Reply:
Agree, fixed